src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.csproj

```diff
@@ -221,5 +231,11 @@
     <TargetingPackReference Include="mscorlib" />
     <TargetingPackReference Include="System.Core" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\System.Security.Cryptography.Algorithms\src\System.Security.Cryptography.Algorithms.csproj">
```

Comment:
We should continue with the plan of putting the Algorithms library changes in first, so that we don't have to do this P2P reference and then immediately patch it back to a package reference.